modules/pages/partials/max-connections.adoc

```diff
@@ -1,4 +1,12 @@
 // tag::limits[]
-IMPORTANT: Each Couchbase Data node allows up to 30,000 concurrent key-value connections xref:server:learn:clusters-and-availability/size-limitations.adoc[by default], in Couchbase Data Platform 6.0 and earlier.
-From 6.5 onwards, the default value is 65,000.
+[IMPORTANT]
+.Connection Limits
+====
+Each Couchbase Data node allows up to 30,000 concurrent key-value connections _per port_ xref:server:learn:clusters-and-availability/size-limitations.adoc[by default], in Couchbase Data Platform 6.0 and earlier.
```

Comment:
Regarding:

 > allows up to 30,000 concurrent key-value connections per port...

Would make more sense to change from "per port" to "per port exposed to user by the Data Service (i.e. 11210 and 11207)".

Reply:
Thanks - made the change.